products/platform/documentation-site/research/search-design-doc.md

```diff
@@ -1,15 +1,157 @@
-# Global Documentation Search
+# Documentation Search MVP
 
-Author: Bill Fienberg
-Last Updated: 2020-01-23
-Reviewers: TBD
+**Author(s):** Bill Fienberg <bill.fienberg@oddball.io>
 
-## Objective
+**Last Updated:** 2020-01-29
 
-## Background
+**Status:** **Draft**
+
+**Approvers:** Andrew G., Megan K., Rian F., DevOps
 
 ## Overview
 
-## Detailed Design
+### Objective
+
+Run our own instance of [Algolia's DocSearch](https://community.algolia.com/docsearch/what-is-docsearch.html) on a custom landing page.
```

Comment:
To me, this makes it sound like we are hosting something besides a webpage and from what I can tell this isn't really the case. We can optionally run the scraper ourselves to have more control of scraping but the index/database is hosted by algolia. All we need to do for searching is insert some javascript into the desired webpage and that piece run queries against algolias API. Do I have that right?

Reply:
Algolia allows people to apply to be included in their scraper that runs every 24 hours, but I don't think we meet all the requirements on their [checklist](https://community.algolia.com/docsearch/who-can-apply.html#the-checklist). 

Therefore, I think running the scraper ourselves is required, not optional.

Reply:
Which of those requirements do you not think we meet @billfienberg? (cc @cvalarida as I noticed you just linked to this comment)

Reply:
Here's the checklist for reference:
https://community.algolia.com/docsearch/who-can-apply.html#the-checklist

I think we fail to satisfy the following requirements:

> You must be the owner of the website, or at least have the power to update its content. You'll have to include a JavaScript snippet to enable DocSearch.

Our docs live in GitHub. We do not own GitHub; Microsoft does. While we have the power to update the contents of our repo, we can't add the JavaScript snippet. 

![no-script-tag-in-github-flavored-markdown](https://user-images.githubusercontent.com/6130520/74054425-abdb6300-49a3-11ea-9cbb-e99fb0376e01.gif)

> Your website must be a documentation website. We do not index blogs or commercial content.

While we have repos that exist specifically for documentation, I don't know if Algolia would consider them a "documentation website".

Reply:
I believe we can put JS snippets on GH pages though? i.e. on https://department-of-veterans-affairs.github.io/va.gov-team/

does the JS snippet need to be on every page, or just _a_ page?

Reply:
The JS snippet will need to be included on every page we want to have our search input. So far a landing page, we'd only need the JS snippet on one page. 

https://community.algolia.com/docsearch/dropdown.html

Here's the actual `DocSearch` source: 
https://github.com/algolia/docsearch/blob/master/src/lib/DocSearch.js

Reply:
@billfienberg I'm confused --- it sounds like you told @meganhkelley  that the checklist requirement we don't meet is "can't add the JS snippet", but if the JS snippet is just about DocSearch and we're putting it on the landing page, then... it sounds like we _do_ meet that checklist requirement?

Can you clarify which checklist requirement we don't meet, and (if it's the JS one) how you're understanding that requirement?